src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius/sylius_promotion.yml

```diff
@@ -14,3 +14,6 @@ sylius_promotion:
             classes:
                 model: Sylius\Component\Core\Model\PromotionCoupon
                 repository: Sylius\Bundle\PromotionBundle\Doctrine\ORM\PromotionCouponRepository
+        catalog_promotion:
+            classes:
+                model: Sylius\Component\Promotion\Model\CatalogPromotion
```

Comment:
Do we need it? It's already set in the `Configuration` file in the `PromotionBundle` and is not (yet) overridden in the `CoreBundle`

Reply:
Yup, this is not needed